src/assets/scripts/client/InputController.js

```diff
@@ -711,6 +731,17 @@ export default class InputController {
     }
 
     /**
+     * Disable default behavior of right-click menu appearing on right click
+     *
+     * @for InputController
+     * @method _disableRightClickMenu
+     * @param event {jQuery event}
+     */
+    _disableRightClickMenu(event) {
+        event.preventDefault();
```

Comment:
do we also need to prevent event bubbling here with `event.stopPropagation();`?

Reply:
We can if that's a thing we should do? Not my area, so you tell me. Though since it is already at the `$body` level, there is basically nowhere for the event to bubble up to I would think? That's not to say we shouldn't put that in there anyway though.

Reply:
looks like we're good here.  disregard for now